ui/src/plugins/vuetify.js

```diff
@@ -28,6 +28,7 @@ export default createVuetify({
           recovery: '#f4bc00',
           completed: '#3fa500',
           failed: '#f41900',
+          canceled: '#616161',
```

Comment:
With this new color we are adding, it's harder to tell the difference between `enqueued` and `canceled`. Maybe we should make `enqueued` or `canceled` lighter? What do you think?